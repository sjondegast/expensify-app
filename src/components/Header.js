import React from "react";
import { NavLink } from 'react-router-dom';

const Header = () => (
    <header>
      <h1>Expensify</h1>
      <NavLink to="/" exact activeClassName="selected">Dashboard</NavLink>
      <NavLink to="/create" activeClassName="selected">Add Expense</NavLink>
      <NavLink to="/edit" activeClassName="selected">Edit Expense</NavLink>
      <NavLink to="/help" activeClassName="selected">Help</NavLink>
    </header>
  );

  export default Header;