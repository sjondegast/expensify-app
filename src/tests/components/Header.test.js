import React from 'react';
import Header from '../../components/Header';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

test('should test something', () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();

});



