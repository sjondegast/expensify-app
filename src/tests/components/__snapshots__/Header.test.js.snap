// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test something 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="selected"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="selected"
    to="/create"
  >
    Add Expense
  </NavLink>
</header>
`;
